docs/how-to/cluster-cfg.rst

```diff
@@ -0,0 +1,60 @@
+Configure cluster_network/ public_network
+=========================================
+
+
+Overview
+--------
+Network configuration is critical for building a high performance Ceph Storage Cluster. Since, Ceph Clients make requests directly to Ceph OSD Daemons. Ceph OSD Daemons perform data replication on behalf of Ceph Clients, which means replication and other factors impose additional loads on Ceph Storage Cluster networks. Refer to `Ceph Network Config <https://docs.ceph.com/en/latest/rados/configuration/network-config-ref/>`_.
+
+Implementation
+--------------
+Microceph cluster config subcommands rely on ‘ceph config’ as the single source of truth for config values and use the same for getting/setting the configs. After updating (setting/resetting) a config value the same request is forwarded to other hosts on the microceph cluster for restarting particular daemons on the host so that updated configs can take effect.
+
+Usage
```

Comment:
I wonder if this section as written is not more of an `Explanation` than a `How-To` in the Diataxis sense

A howto should be practical and for work -- i.e. describe steps to accomplish a practical work task. While the section describes in general how to configure MicroCeph

Think it would probably be good to have both a how-to to describe splitting up public and cluster net while also having a more generic explanation on configuring MicroCeph

Reply:
+1 :D